CHANGELOG.md

```diff
@@ -23,10 +23,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 ### Changed
 
-- Changed Cat Claws recipe to Makeshift Claws || Iron Ingot
+- Changed Cat Claws recipe to Makeshift Claws && Iron Ingot
 - Updated Cat Claws texture
+- Changed Jousting Lance recipe to Wooden Lance && Iron Ingot
+- Changed Cigarette Lance recipe to Regilance && Eightball
+- Rebalanced Cigarette Lance Grist Cost and Damage
 - Added alchemy recipe for Ancient Debris: Steel Beam && Raw Uranium
 - Added Grist Cost to Eightball and Dice items
+- Changed Applescab Scalemate's recipe to Red Wool && Dragon Breath
+- Added alchemy recipes for every Scalemate
+- Changed what dyes can be used to change Scalemate color in the offhand to reflect recipe additions
+- Changed Pinesnort Scalemate's registry from Pinesnout to Pinesnort
```

Comment:
Change this from registry to something along the lines of "in-game name" and this PR looks good to merge!

Reply:
:33 < oops yeah furgot to edit that part of the log!!